src/PinPatternManager.cpp

```diff
@@ -6,15 +6,9 @@
 
 const char* const TAG = "PinPatternManager";
 
-struct Pattern {
-  unsigned int pin;
-  OpenShock::PinPatternManager::State* pattern;
-  std::size_t patternLength;
-};
-
```

Comment:
Was this just unused?
Didn't notice xD

Reply:
Hah, yeah, didn't find any references to it at all.